api/test_liveness_mtchoun-mouh.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Selenium IDE
+import pytest
+import time
+import json
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.webdriver.chrome.options import Options
+
+class TestLiveness():
+  def setup_method(self, method):
+    chrome_options = Options()
+    chrome_options.add_argument('--headless')
+    chrome_options.add_argument('--no-sandbox')
+    chrome_options.add_argument('--disable-dev-shm-usage')
+    self.driver = webdriver.Chrome(chrome_options=chrome_options)
+    self.vars = {}
+  
+  def teardown_method(self, method):
+    self.driver.quit()
+  
+  def test_liveness(self):
+    # Test name: liveness
+    # Step # | name | target | value | comment
+    # 1 | open | / |  | 
+    self.driver.get("https://mtchoun-mouh.mongulu.cm/")
```

Comment:
The website is not good here. You should check the website from an env variable as for eah branch we will have a new website. You can get it in deploy_push.yml https://github.com/mongulu-cm/mtchoun-mouh/blob/40bf4625e79652b750c1a8daf87b8874fe6ea58c/.github/workflows/deploy_push.yml#LL82-L86C92 & it's used on check readiness https://github.com/mongulu-cm/mtchoun-mouh/blob/40bf4625e79652b750c1a8daf87b8874fe6ea58c/.github/workflows/deploy_push.yml#L102